mate-panel/panel-action-button.c

```diff
@@ -213,13 +218,202 @@ panel_action_logout (GtkWidget *widget)
 						      PANEL_SESSION_MANAGER_LOGOUT_MODE_NORMAL);
 }
 
+/* Shutdown
+ */
+#ifdef HAVE_WAYLAND
 static void
-panel_action_shutdown (GtkWidget *widget)
+show_error_dialog(GtkWidget *label)
 {
-	PanelSessionManager *manager;
+	GtkWidget *dialog, *box, *content_area, *button, *image;
+	dialog = gtk_dialog_new();
+	button = gtk_dialog_add_button (GTK_DIALOG(dialog),
+			       "Cancel",
+			       GTK_RESPONSE_CANCEL);
+
+	image = gtk_image_new_from_icon_name("process-stop", GTK_ICON_SIZE_MENU);
```

Comment:
A space is missing here and some other places.
```suggestion
	image = gtk_image_new_from_icon_name ("process-stop", GTK_ICON_SIZE_MENU);
```

Reply:
Found and fixed in the latest force-push